provisioner/data.js

```diff
@@ -261,9 +253,8 @@ WorkerType.prototype.json = function () {
  * and optionally a single property from it.
  */
 WorkerType.prototype.getInstanceType = function (instanceType) {
-  var types = this.instanceTypes.filter(function (t) {
-    return t.instanceType === instanceType;
-  });
+  let types = this.instanceTypes.filter(t => t.instanceType === instanceType);
```

Comment:
Tip: lodash has `_.filter(this.instanceTypes, {instanceType: instanceType})` which can also be written as:
`_.filter(this.instanceTypes, {instanceType})` because key and variable name for value is the same.

Just a tip, I haven't found many places to use this yet...